lib/macos.js

```diff
@@ -19,9 +19,25 @@ const parseMac = stdout => {
 	}
 };
 
-module.exports = async () => {
-	const {stdout} = await execFile(bin);
+const getArguments = options => {
+	if (!options) {
+		return [];
+	}
+
+	const args = [];
+	if (options.screenRecordingPermission === false) {
+		args.push('--disable-screen-recording-permission');
```

Comment:
```suggestion
		args.push('--no-screen-recording-permission');
```

`--no` is the convention for inverted flags.